Signal/src/ViewControllers/MediaDetailViewController.m

```diff
@@ -714,7 +714,7 @@ - (void)didPressPauseBarButton:(id)sender
 
 #pragma mark - Presentation
 
-- (void)presentFromViewController:(UIViewController *)viewController replacingView:(UIView *)view;
```

Comment:
This formatting glitch appears often in our codebase - I've fixed it everywhere that I can.